shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1847,9 +1853,29 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer*)gestureRecognizer
   return YES;
 }
 
+- (BOOL)gestureRecognizer:(UIGestureRecognizer*)gestureRecognizer
+       shouldReceiveEvent:(UIEvent*)event API_AVAILABLE(ios(13.4)) {
+  if (gestureRecognizer == _continuousScrollingPanGestureRecognizer &&
+      event.type == UIEventTypeScroll) {
```

Comment:
do we need this `UIEventTypeScroll` check? What other event type is possible?

Reply:
UIEventTypeTouches will be processed here, we need to make sure it is this scroll event that follows after use of the continuous pan gesture recognizer.